t/e2e/api/backend_test.go

```diff
@@ -28,6 +28,9 @@ type APIBackendTestSuite struct {
 // FIXME(tiabc): There's also a test with the same name in geth/node/manager_test.go
 // so this test should only check StatusBackend logic with a mocked version of the underlying StatusNode.
 func (s *APIBackendTestSuite) TestRaceConditions() {
+	if GetNetworkID() == params.MainNetworkID {
```

Comment:
There are some calls to `CompleteTransaction` here but it looks like they are not meant to be successful. Does it fail when run on mainnet?

Reply:
Good catch, works with mainnet.